PowerEditor/src/NppDarkMode.h

```diff
@@ -118,7 +118,9 @@ namespace NppDarkMode
 	void setAdvancedOptions();
 
 	bool isWindows10();
+	bool isWindows10(DWORD build);
 	bool isWindows11();
+	bool isWindows11(DWORD build);
```

Comment:
This function is never called. It shouldn't be here, should it?

Reply:
I've added it for convenience, currently it isn't used, but future Windows 11 builds might have new API for dark mode.

It can be removed.